MobalMobal/MobalMobal/Main/MainViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  MainViewController.swift
+//  MobalMobal
+//
+//  Created by 김재희 on 2021/02/27.
+//
+
+import SnapKit
+import Then
+import UIKit
+
+class MainViewController: UIViewController {
+    // MARK: - UIComponent
+    let titleView: UIView = {
+        let view: UIView = UIView(frame: .zero)
+        view.backgroundColor = .backgroundColor
+        return view
+    }()
+    
+    let titleLabel: UILabel = {
+        let label: UILabel = UILabel(frame: .zero)
+        label.text = "Hi, jaehui"
+        label.font = UIFont(name: "Futura-Bold", size: 30)
+        label.textColor = .white
+        return label
+    }()
+    
+    let notiListButton: UIButton = {
+        let button: UIButton = UIButton(frame: .zero)
+        button.setImage(UIImage(named: "icMyProfile"), for: .normal)
+        return button
+    }()
+    
+    let profileImageView: UIImageView = {
+        let imageView: UIImageView = UIImageView(frame: .zero)
+        imageView.image = UIImage(named: "icAlarm")
+        imageView.contentMode = .scaleAspectFill
+        return imageView
+    }()
+    
+    let tableView: UITableView = {
+        let tableView: UITableView = UITableView(frame: .zero, style: .plain)
+        tableView.backgroundColor = .backgroundColor
+        tableView.separatorStyle = .none
+        tableView.tableFooterView = UIView()
+        tableView.separatorInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
```

Comment:
```suggestion
        tableView.separatorInset = .zero
```

이렇게 표현할 수도 있을 것 같습니다.